docs/reference-architectures/app-service-web-app/baseline-zone-redundant-content.md

```diff
@@ -38,16 +38,16 @@ Network security is at the core of the App Services baseline architecture (*see
 ### Network flows
 
 :::image type="complex" source="images/baseline-app-service-network-architecture.svg" lightbox="images/baseline-app-service-network-architecture.svg" alt-text="Diagram that shows a baseline App Service network architecture.":::
-    The diagram is the same as the Baseline Azure App Service architecture with two numbered network flows. The inbound flow shows a line from the user to the Azure Application Gateway with Web Application Firewall (WAF). The second number is for WAF. The third number shows that private DNS zones are linked to the virtual network. The fourth number shows App Gateway using private endpoints to communicate to App Service. The first number in the flow from App Service to Azure PaaS services shows an arrow from App Service to a virtual interface. The second, again shows that private DNS zones are linked to the virtual network. The third shows arrows from the virtual interface communicating via private endpoints to Azure PaaS services.
+    The diagram resembles the Baseline Azure App Service architecture with two numbered network flows. The inbound flow shows a line from the user to the Azure Application Gateway with Web Application Firewall (WAF). The second number is for WAF. The third number shows private DNS zones are linked to the virtual network. The fourth number shows App Gateway using private endpoints to communicate with App Service. The first number in the flow from App Service to Azure PaaS services shows an arrow from App Service to a virtual interface. The second shows that private DNS zones are linked to the virtual network. The third shows arrows from the virtual interface communicating via private endpoints to Azure PaaS services.
 :::image-end:::
 *Figure 2: Network architecture of the baseline Azure App Service application*
 
-The following are descriptions of the inbound flow of internet traffic to the App Service instance, and the flow from the App Service to Azure services.
+The following are descriptions of the inbound flow of internet traffic to the App Service instance and the flow from the App Service to Azure services.
```

Comment:
I don't think there is an issue with the comma.

Reply:
Updated.